VERSION

```diff
@@ -1 +1 @@
-1.0
+1.1
```

Comment:
As above, this should be 1.0.1 (I use the first two parts to denote compatibility with gap-packages/Memoisation but the third part is fine to change).

Reply:
Ah ok. I had just seen only the first two parts in "CHANGES" so I assumed you prefered that. I think 1.0.1 is better.